libs/ui/src/session_time_limit_tracker.tsx

```diff
@@ -148,6 +153,7 @@ interface SessionTimeLimitTrackerProps {
     | DippedSmartCardAuth.AuthStatus
     | InsertedSmartCardAuth.AuthStatus;
   logOut: () => void;
+  systemSettings?: SystemSettings | null;
```

Comment:
The breakdown is:

- `undefined` ➡️ Query hasn't finished yet
- `null` ➡️ Query has finished, and there's no system settings object because the machine hasn't been configured yet
- `SystemSettings` ➡️ Query has finished, and there is a system settings object

We could alternatively have the backend fall back to `DEFAULT_SYSTEM_SETTINGS` when there's no system settings object, but with the current approach of returning `null`, we do get consistency with how the app backends handle the election definition.

There's a little more on the distinction between `undefined` and `null` in react-query in this [comment thread](https://github.com/votingworks/vxsuite/pull/3278#discussion_r1165647415)

Reply:
Cool, thanks!

I think we should be biasing towards setting defaults in the backend to keep business logic colocated (and make testing easier).

Reply:
Yeah thinking some more, I agree. Let me try updating and if it doesn't require fixing too many tests 😅, I'll incorporate into this PR

Reply:
Wasn't bad at all! https://github.com/votingworks/vxsuite/pull/3385/commits/b154c952566b38f4e14857a8a9eb6cb880005cd3

Reply:
Awesome!